### ISSUE TITLE
Anonymous function does not work

### ISSUE THREAD

**Issue by thomas9911** [thomas9911/erlang-term#1]
in elixir:

```elixir
iex> f = fn x -> x + 2 end
iex> :erlang.term_to_binary(f) |> IO.inspect(limit: 100000); :ok
<<131, 112, 0, 0, 0, 192, 1, 76, 80, 224, 147, 120, 219, 253, 166, 238, 44, 96,
  86, 8, 53, 142, 187, 0, 0, 0, 44, 0, 0, 0, 1, 100, 0, 8, 101, 114, 108, 95,
  101, 118, 97, 108, 97, 44, 98, 2, 98, 135, 4, 88, 100, 0, 13, 110, 111, 110,
  111, 100, 101, 64, 110, 111, 104, 111, 115, 116, 0, 0, 0, 106, 0, 0, 0, 0, 0,
  0, 0, 0, 104, 4, 116, 0, 0, 0, 0, 100, 0, 4, 110, 111, 110, 101, 100, 0, 4,
  110, 111, 110, 101, 108, 0, 0, 0, 1, 104, 5, 100, 0, 6, 99, 108, 97, 117, 115,
  101, 97, 1, 108, 0, 0, 0, 1, 104, 3, 100, 0, 3, 118, 97, 114, 97, 1, 100, 0,
  4, 95, 120, 64, 49, 106, 106, 108, 0, 0, 0, 1, 104, 5, 100, 0, 2, 111, 112,
  97, 1, 100, 0, 1, 43, 104, 3, 100, 0, 3, 118, 97, 114, 97, 1, 100, 0, 4, 95,
  120, 64, 49, 104, 3, 100, 0, 7, 105, 110, 116, 101, 103, 101, 114, 97, 0, 97,
  2, 106, 106>>
:ok
```

In rust:

```rust 
    let data = [131, 112, 0, 0, 0, 192, 1, 76, 80, 224, 147, 120, 219, 253, 166, 238, 44, 96,
    86, 8, 53, 142, 187, 0, 0, 0, 44, 0, 0, 0, 1, 100, 0, 8, 101, 114, 108, 95,
    101, 118, 97, 108, 97, 44, 98, 2, 98, 135, 4, 88, 100, 0, 13, 110, 111, 110,
    111, 100, 101, 64, 110, 111, 104, 111, 115, 116, 0, 0, 0, 106, 0, 0, 0, 0, 0,
    0, 0, 0, 104, 4, 116, 0, 0, 0, 0, 100, 0, 4, 110, 111, 110, 101, 100, 0, 4,
    110, 111, 110, 101, 108, 0, 0, 0, 1, 104, 5, 100, 0, 6, 99, 108, 97, 117, 115,
    101, 97, 1, 108, 0, 0, 0, 1, 104, 3, 100, 0, 3, 118, 97, 114, 97, 1, 100, 0,
    4, 95, 120, 64, 49, 106, 106, 108, 0, 0, 0, 1, 104, 5, 100, 0, 2, 111, 112,
    97, 1, 100, 0, 1, 43, 104, 3, 100, 0, 3, 118, 97, 114, 97, 1, 100, 0, 4, 95,
    120, 64, 49, 104, 3, 100, 0, 7, 105, 110, 116, 101, 103, 101, 114, 97, 0, 97,
    2, 106, 106];

    let x = erlang_term::RawTerm::from_bytes(&data);
    // x is an error (EOF)
```

**Comment by yangcancai** [comment]
I fix it here #5

**Comment by thomas9911** [comment]
Fixed in 0.2.2
